README.md

```diff
@@ -45,7 +46,17 @@ To see a list of linters included for each configuration:
 names(lintr::linters_with_defaults())
 
 # full
-names(lintr::linters_with_tags(tags = NULL))
+names(lintr::all_linters())
+```
+
+You can use helper functions from `{usethis}` package to generate GitHub action workflow files:
+
+```R
+# in a project:
+usethis::use_github_action("lint-project")
```

Comment:
Why split this off?

Reply:
Separation of concerns. 

Not every lintr user will concern themselves with integrating the workflow with GitHub actions (many might not even be using GitHub), so we should separate the documentation of how to set up config file from how to use lintr with GHA.

Reply:
Sounds fair, should we add a subsection then?
`### Setting up GitHub Actions` maybe?